app/styles/themes/gothamist/_theme.variables.scss

```diff
@@ -96,11 +96,11 @@
   --max-width:      1024px;
   --max-width-xl:   1130px;
   --heading-height: 72px;
-  
+
   // Grid
-  --col-width: 48.5px;
```

Comment:
I think this is still used in `_layout.wrappers.scss`. Is there going to be a follow up to this PR to work on the layout containers there?